src/lib/y2storage/proposal_settings.rb

```diff
@@ -644,6 +655,7 @@ def ng_string_representation
       "    delete_resize_configurable: #{delete_resize_configurable}\n" \
       "    lvm_vg_strategy: #{lvm_vg_strategy}\n" \
       "    lvm_vg_size: #{lvm_vg_size}\n" \
+      "    multidisk_first: #{multidisk_first}\n" \
```

Comment:
NP: maybe we should start thinking about using pretty print instead of manually fomatting the output.

Reply:
If I remember correctly, it was discussed during the planning at it will be addressed in the ["Better logging of proposal settings"](https://trello.com/c/tYqIvWA4/1211-1-sle15-sp1-storage-ng-better-logging-of-proposal-settings) PBI.

Reply:
Oh, right! Thanks! As you see, I have bad memory.

Reply:
Don't worry. Mine is worst :cry: